sources/pmwebapi_test.go

```diff
@@ -0,0 +1,95 @@
+package sources
+
+import (
+	"testing"
+	"encoding/json"
+)
+
+type AutoGenerated struct {
+	Args struct {
+	} `json:"args"`
+	Headers struct {
+		Accept string `json:"Accept"`
+		AcceptEncoding string `json:"Accept-Encoding"`
+		AcceptLanguage string `json:"Accept-Language"`
+		CacheControl string `json:"Cache-Control"`
+		Connection string `json:"Connection"`
+		Host string `json:"Host"`
+		UpgradeInsecureRequests string `json:"Upgrade-Insecure-Requests"`
+		UserAgent string `json:"User-Agent"`
+		XBluecoatVia string `json:"X-Bluecoat-Via"`
+	} `json:"headers"`
+	Origin string `json:"origin"`
+	URL string `json:"url"`
+}
+
+func TestGetRequest(t *testing.T) {
+	testString := "http://httpbin.org/get"
+	response := getRequest(testString)
+	if response == nil {
+		t.Fatal("get request not completed")
```

Comment:
You should write these error messages like sentences. Capitalized first letter, period...that sort of thing. Also, it can be helpful to communicate what you got vs. what you expected. See: https://github.com/HewlettPackard/lustre_exporter/blob/master/sources/procfs_test.go#L40